openvdb_points/tools/AttributeArray.cc

```diff
@@ -358,6 +358,18 @@ AttributeArray::setHidden(bool state)
 }
 
 
+void
+AttributeArray::setString(bool state)
```

Comment:
In the groups pull request, we settled on not exposing a method like this on the AttributeArray base class, and instead, made this local to the AttributeGroup class. It would be good to have both the AttributeGroup and the AttributeArrayString follow the same pattern here...